scripts/ci/release.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dir=$(dirname $0)
+projectRoot=${dir}/../..
+RUNTIME_NAME='tomcat'
+RUNTIME_VERSION='8.5'
+
+if [ -z "${DOCKER_TAG}" ]; then
+  DOCKER_TAG="${RUNTIME_VERSION}-$(date -u +%Y-%m-%d_%H_%M)"
+fi
+
+if [ -z "$DOCKER_NAMESPACE" ]; then
+  DOCKER_NAMESPACE="gcr.io/$(gcloud info \
```

Comment:
Indeed as more functionalities using the gcloud cli may appear I added a new library (utils/gcloud.sh)